
<p align="center">
	<img src="https://github.com/sergiodealencar/sergiodealencar/blob/master/banner.png">
</p>

# Sérgio de Alencar
<sub>*Bioinformatics Researcher* at the Catholic University of Brasília</sub>

Sérgio possui graduação em Ciências Biológicas com ênfase em Genética pela University of Leicester (2002), mestrado em Bioinformática pela Katholieke Universiteit Leuven (2004), doutorado em Bioinformática pela Universidade Federal de Minas Gerais (UFMG) e pós-doutoramento na Embrapa Recursos Genéticos e Biotecnologia, atuando na área de Genômica Computacional com análises de dados de sequenciamento de segunda geração. Atualmente é professor/pesquisador na área de Bioinformática no curso de Pós-Graduação em Ciências Genômicas e Biotecnologia na Universidade Católica de Brasília.

**Background in:** Python

**Links:**
* [LinkedIn](https://bit.ly/2QO3nLd)
* [Medium](https://medium.com/@sergiodealencar)
* [Currículo Lattes](http://lattes.cnpq.br/7981398735404589)


## Projetos:
Veja os tutoriais publicados do meu Portfolio:

* **Frequency Calculations of Autosomal Recessive Disease using Hardy-Weinberg Equations** (https://bit.ly/32QnK02) (Descrição no Medium)


## Artigos Científicos:

* Benchmarking analysis of deleterious SNP prediction tools on CYP2D6 enzyme (2020) 
* Genome-wide analysis of the transcriptional response to drought stress (2020) (https://bit.ly/2IxsG3g)
* A preliminary Analysis of the Impact of Missense Variants of Unknown Significance in the APOA1 Gene using prediction softwares (2020)
* Spring is coming: genetic analyses of the bud break date locus reveal candidate genes from the cold perception pathway to dormancy release in apple (Malus × domestica Borkh.) (2019) (https://bit.ly/31eGF4s)
* Deciphering the structural basis for glucocorticoid resistance caused by missense mutations in the ligand binding domain of glucocorticoid receptor (2019) (https://bit.ly/37fksHr)
* Transcriptome analysis and gene networks in a rare pediatric tumor (2019) (https://bit.ly/2H2fHXc)
* Comparative transcriptome analyses of magainin I-susceptible and I-resistant Escherichia coli strains (2018) 
* Theoretical structural characterization of lymphoguanylin: a potential candidate for the development of drugs to treat gastrointestinal disorders (2017) (https://bit.ly/3lOfSnE)
* In silico analyses of deleterious missense SNPs of human apolipoprotein E3 (2017) (https://bit.ly/3lVqtNx)
* Computational investigation of growth hormone receptor Trp169Arg heterozygous mutation in a child with short stature (2017) (https://bit.ly/3dBmhzz)
* Genomic Comparison among Lethal Invasive Strains of Streptococcus pyogenes Serotype M1 (2017) (https://bit.ly/2T4fULO)
* Venom gland transcriptome analyses of two freshwater stingrays (Myliobatiformes: Potamotrygonidae) from Brazil (2016) (https://bit.ly/345c0bK)
* Prediction of the Impact of Coding Missense and Nonsense Single Nucleotide Polymorphisms on HD5 and HBD1 Antibacterial Activity against Escherichia coli (2016)
* Draft Genome Sequence of a Novel Member Isolated from Brazilian Amazon Soil (2016)
* Structural impact analysis of missense SNPs present in the uroguanylin gene by long-term molecular dynamics simulations (2016)
* Computational Analyses and Prediction of Guanylin Deleterious SNPs (2015)
* HD5 and HBD1 Variants' Solvation Potential Energy Correlates with their Antibacterial Activity against Escherichia coli (2015)
* Shedding some light over the floral metabolism by arum lily (Zantedeschia aethiopica) spathe de novo transcriptome assembly (2014)
* A Comprehensive In Silico Analysis of the Functional and Structural Impact of SNPs in the IGF1R Gene (2010)
